### DISCUSSION TITLE
Ability to remove a page from public site

### DISCUSSION THREAD

**Discussion by xavier-chiliz** [GitbookIO/community#423] [Feature Requests]
# Current state

As per your team, it is not currently possible to unpublish a single page, just a whole space.

To move a single page out of space that is published, one would have to:

- copy and paste it into a draft Change Request (and remove it from main version)
- copy and paste it into other, unpublished space

# Issue with current state

Having to manually copy-paste content left and right is really cumbersome; it would be great to be able to take a page "backstage" without having to rely on change request through temporary page deletion. Let alone having to do that on several pages or a whole content section.

# Proposed solution

It would be nice to have any of these:
- a "Draft" flag, 
- a "Private/Public" setting 
- a "Published/Unpublished" (see [request 22](https://github.com/orgs/GitbookIO/discussions/22)) or "Hide" toggle (see [request 12](https://github.com/orgs/GitbookIO/discussions/12)).
- The ability to copy/move a page from one (public) space to another (private) one (see [request 21](https://github.com/orgs/GitbookIO/discussions/21)).

I hope the team can see the relevance of such a feature for a knowledge management tool.

**Comment by xavier-chiliz** [comment]
Updated the request with a reference to https://github.com/orgs/GitbookIO/discussions/21.

Is there anything happening around this kind of idea? 
It seems to be quite a big deal, seeing how many requests mention it.

**Comment by MadeByPP** [comment]
It would be nice to be able to set private/public settings for each group.
For example, “Help (public)”, “API (public)”, “Document for admin only (private)”

**Comment by samvandamme** [comment]
This would be very handy for us too. We create integrations with other companies and the documentation is mostly ready DAYS before we can officially launch the integration. Would be amazing to have a single page unpublished until we really want to publish it.

**Comment by chris-habasinski** [comment]
Hi, I am just a user, new user at that, so maybe I am missing something. If you use the github sync integration and provide your own SUMMARY.md file, you can have the new page in the repo but it will not appear in the published site until you include it in the SUMMARY.md file.
I manage my SUMMARY.md file with a script to generate the contents. Using a filenaming convention like "hide.new-page.md" would allow the script to skip these files when creating the summary. This could of course be done by hand if the volumes are relatively low.
HTH
Chris

**Comment by xavier-chiliz** [comment]
Closing this as it seems to be resolved elegantly with the "Hide page" feature presented in #675 .

**Comment by xavier-chiliz** [comment]
Re-opening the idea because of this:

> Your users can still access hidden pages via direct link, find them through an on-site search or by asking GitBook AI — and they’re indexed by search engines. 

Clearly it's a miss. There _is_ still a need to "unpublish" a page temporarily.

**Reply by aikoii** [reply]
Hey @xavier-chiliz! We’ve just shipped a feature that addresses this 🎉. You can find more details in #730. Thanks again everyone for your feedback 👋
